string-manipulation-1/src/get-first-char.js

```diff
@@ -0,0 +1,13 @@
+/* exported getFirstChar */
+function getFirstChar(string) {
+  var firstCharIs = ' ';
+  for (var i = 0; i < 1; i++) {
+    firstCharIs = string[i];
+  }
+  return firstCharIs;
+}
```

Comment:
Instead of looping, you can just select the first index! 
```
function getFirstChar(string) {
  return string[0];
}
```

Reply:
Great tips.  Thanks Shawn!